### PR TITLE
refactor: Identify refactor points in Identity related workflows

### PR DESCRIPTION
**Pull request by alexs-mparticle**
## Instructions
 1. PR target branch should be against `development`
 2. PR title name should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-title-check.yml
 3. PR branch prefix should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-branch-check-name.yml

 ## Summary
 - Identify future refactor points for Identity related workflows

 ## Testing Plan
 - [ ] Was this tested locally? If not, explain why.
 - N/A - this only adds comments to certain parts of the code

 ## Reference Issue (For mParticle employees only.  Ignore if you are an outside contributor)
 - Closes https://go.mparticle.com/work/SQDSDKS-6486

### CODE REVIEW

File: src/identity.js [hunk 1]
```diff
@@ -796,6 +801,7 @@ export default function Identity(mpInstance) {
         },
     };
 
+    //go.mparticle.com/work/SQDSDKS-6354
```
Comment (rmi22186) [inline src/identity.js:804]:
```suggestion
    // https://go.mparticle.com/work/SQDSDKS-6354
```

File: src/identity.js [hunk 2]
```diff
@@ -1791,6 +1803,7 @@ export default function Identity(mpInstance) {
         mpid,
         prevUserIdentities
     ) {
+        // https://go.mparticle.com/work/SQDSDKS-6354
```
Comment (rmi22186) [inline src/identity.js:1806]: Going to this work item doesn't match the code block here.

File: src/sdkRuntimeModels.ts [hunk 3]
```diff
@@ -34,7 +34,10 @@ export interface SDKEvent {
     IsFirstRun: boolean;
     EventName: string;
     EventCategory: number;
+
+    // https://go.mparticle.com/work/SQDSDKS-5196
     UserAttributes?: { [key: string]: string | string[] | null };
+
```
Comment (rmi22186) [inline src/sdkRuntimeModels.ts:40]: is this whitespace intentional?

File: src/serverModel.ts [hunk 4]
```diff
@@ -46,7 +46,10 @@ export interface IServerV2DTO {
     dp_v?: number;
     n?: string;
     et?: number;
+
+    // https://go.mparticle.com/work/SQDSDKS-5196
     ua?: Dictionary<string | string[]>;
+
```
Comment (rmi22186) [inline src/serverModel.ts:52]: is this whitespace intentional?

**Review by rmi22186 — CHANGES_REQUESTED** [review]

File: src/sdkRuntimeModels.ts (hunk 3 above)
Comment (alexs-mparticle) [reply src/sdkRuntimeModels.ts:40]: Yes. It's to show that the comment only applies to that specific type definition.

File: src/serverModel.ts (hunk 4 above)
Comment (alexs-mparticle) [reply src/serverModel.ts:52]: Yes. It's to show that the comment only applies to that specific type definition.

**Review by rmi22186 — APPROVED** [review]

**Pull request merged**